pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/LeafStageTransferableBlockOperator.java

```diff
@@ -176,16 +174,25 @@ protected TransferableBlock getNextBlock()
     // Terminate when receiving exception block
     Map<Integer, String> exceptions = _exceptions;
     if (exceptions != null) {
```

Comment:
This block is used twice so it could be made into a regular or static constructor in ErrorMseBlock.

Reply:
I've done a different thing. I've created that static method in QueryErrorCode.